src/components/Store.react.js

```diff
@@ -88,7 +49,13 @@ class WebStore {
     }
 
     getItem(key) {
-        return JSON.parse(this._storage.getItem(key));
+        try {
+            return JSON.parse(this._storage.getItem(key));
+        } catch (e) {
+            // in case we somehow got a non-JSON value in storage,
+            // just ignore it.
+            return null;
```

Comment:
closes #456

Reply:
We could also try and resolve the `undefined` case gracefully and only resort to the `catch` if it's a truly unexpected value.

```
const value = this._storage.getItem(key);
return isNil(value) ? null : JSON.parse(value);
```

Reply:
The problem I was hitting is sometimes the string `’undefined’` was making its way into storage. That’s neither nil nor JSON parsable. Shouldn’t happen anymore, but if an app running an old version of this component corrupts the storage, we have no way to recover. I suppose we could make a narrow exception for just the string I’ve seen but I didn’t see the harm in catching more general corruption.

Reply:
No worries. Just nitpicking. I agree there's no scenario where we dont `try..catch`, might as well keep it as-is.